Dockerfile

```diff
@@ -19,11 +18,10 @@ COPY ./db/wait-for /app/wait-for
 
 ADD ./src/main/resources/template /app/template
 
-RUN chmod +x /app/wait-for
-RUN chmod +x /app/verify/verify
+RUN chmod +x /app/wait-for /app/verify/verify  \
+    && apt-get update && apt-get install -y --no-install-recommends netcat  \
```

Comment:
By default, Ubuntu installs recommended but not suggested packages. With `--no-install-recommends`, only the main dependencies (packages in the `Depends` field) are installed.